🐛 Bug: emoji-blast types incorrectly require initialVelocities to be filled out

### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/emoji-blast/issues?q=is%3Aissue) and found none that matched my issue.

### Expected

The following should be allowed:

```ts
emojiBlast({
	physics: {
		gravity: -0.35,
		initialVelocities: {
			y: { max: 14, min: 11.7 },
		},
	},
});
```

### Actual

```plaintext
error TS2739: Type '{ y: { max: number; min: number; }; }' is missing the following properties from type 'InitialVelocities': rotation, x

21    initialVelocities: {
      ~~~~~~~~~~~~~~~~~
```

### Additional Info

Splitting out of #283.

--- a/packages/emoji-blast/src/emojiBlast.ts
+++ b/packages/emoji-blast/src/emojiBlast.ts
@@ -8,7 +8,7 @@ import { animate } from "./animate.js";
 import { defaultEmojis } from "./emojis.js";
 import { EmojiEvents, initializeEvents } from "./events.js";
 import { createStyleElementAndClass } from "./styles.js";
-import { obtainValue, shuffleArray } from "./utils.js";
+import { MakePartial, obtainValue, shuffleArray } from "./utils.js";
 
 /**
  * Settings to launch a blast of emojis! 🎆
@@ -42,7 +42,7 @@ export interface EmojiBlastSettings {
 	/**
 	 * Runtime change constants for emoji element movements.
 	 */
-	physics: Partial<EmojiPhysics>;
+	physics: MakePartial<EmojiPhysics, "initialVelocities">;
 
 	/**
 	 * How to determine where to place blasts of emojis around the page.

--- a/packages/emoji-blast/src/utils.ts
+++ b/packages/emoji-blast/src/utils.ts
@@ -1,3 +1,8 @@
+export type MakePartial<T, PartialKeys extends keyof T> = Partial<
+	Omit<T, PartialKeys>
+> & {
+	[K in PartialKeys]?: Partial<T[K]>;
+};
 /**
  * Grabs the value of an item or item-returning function.
  * @param value   Item or item-returning function.

--- a/packages/emoji-blast/src/utils.ts
+++ b/packages/emoji-blast/src/utils.ts
@@ -1,3 +1,6 @@
+/**
+ * Like Partial, but also makes particular properties Partial one level deep.
+ */
 export type MakePartial<T, PartialKeys extends keyof T> = Partial<
 	Omit<T, PartialKeys>
 > & {

